dev/Scroller/Scroller.cpp

```diff
@@ -4752,9 +4752,9 @@ void Scroller::UpdateManipulationRedirectionMode()
 {
     if (m_scrollerVisualInteractionSource)
     {
-        winrt::InputKind inputKind = InputKind();
+        winrt::InputKind ignoredInputKind = IgnoredInputKind();
 
-        if ((inputKind & winrt::InputKind::MouseWheel) == winrt::InputKind::MouseWheel)
+        if ((ignoredInputKind & winrt::InputKind::MouseWheel) != winrt::InputKind::MouseWheel)
```

Comment:
>gnoredInputKind & winrt::InputKind::MouseWheel) != winrt::InputKind::MouseWheel [](start = 14, length = 79)

nit: It would make it easier to read the code if you can create some helpers like IsMouseWheelIgnored, IsTouchIgnored etc.